ros/lib/config.py

```diff
@@ -25,3 +25,8 @@
 APP_NAME = os.getenv("APP_NAME", "ros")
 
 INSIGHTS_EXTRACT_LOGLEVEL = os.getenv("INSIGHTS_EXTRACT_LOGLEVEL", "ERROR")
+
+# TODO
+RBAC_HOST = os.getenv("RBAC_HOST", "localhost")
+RBAC_PORT = os.getenv("RBAC_PORT")
+RBAC_SVC_URL = f"http://{RBAC_HOST}:{RBAC_PORT}/"
```

Comment:
does this URL become invalid if RBAC_PORT is not set?

Reply:
missed adding the port, it will default to 8114 now.